k8s/helm-charts/seldon-core-v2-runtime/templates/seldon-runtime.yaml

```diff
@@ -55,4 +55,5 @@ spec:
     tracingConfig:
       disable: {{ .Values.config.tracingConfig.disable }}
       otelExporterEndpoint: {{ .Values.config.tracingConfig.otelExporterEndpoint }}
+      otelExporterProtocol: {{ .Values.config.tracingConfig.otelExporterProtocol }}
```

Comment:
@RafalSkolasinski following this change, should we update any of the helm chart versions? I don't know how helm chart version updates are handled, or if there is a policy on updating them. Atm, for example, I've seen that the appVersion has nothing to do with the Seldon Core version. Happy to follow what you suggest here.